adhocracy4/follows/static/follows/FollowButton.jsx

```diff
@@ -1,77 +1,67 @@
+import django from 'django'
+import React, { useState, useEffect } from 'react'
+import Alert from '../../../static/Alert'
+
 const api = require('../../../static/api')
-const django = require('django')
-const React = require('react')
-const Alert = require('../../../static/Alert')
 
-class FollowButton extends React.Component {
-  constructor (props) {
-    super(props)
+export const FollowButton = (props) => {
+  const [followed, setFollowed] = useState(undefined)
+  const [alert, setAlert] = useState(null)
 
-    this.state = {
-      followed: undefined,
-      follows: 0,
-      alert: null
-    }
+  const removeAlert = () => {
+    setAlert(null)
   }
 
-  removeAlert () {
-    this.setState({
-      alert: null
-    })
-  }
+  const followDescriptionText = followed
+    ? django.gettext('Click to no longer be updated about this project via email.')
+    : django.gettext('Click to be updated about this project via email.')
 
-  toggleFollow () {
-    let followAlertText
-    if (this.state.followed) {
-      followAlertText = django.gettext('You will no longer be updated via email.')
-    } else {
-      followAlertText = django.gettext('You will be updated via email.')
-    }
-    api.follow.change({ enabled: !this.state.followed }, this.props.project)
+  const toggleFollow = () => {
+    const followAlertText = followed
```

Comment:
maybe group them with the other translations above so they are all in one place?